js_modules/dagit/packages/core/src/app/FallthroughRoot.tsx

```diff
@@ -85,11 +85,11 @@ const FinalRedirectOrLoadingRoot = () => {
     <Box padding={{vertical: 64}}>
       <NonIdealState
         icon="no-results"
-        title={repoWithNoJob ? 'No jobs' : 'No repositories'}
+        title={repoWithNoJob ? 'No jobs' : 'No definitions'}
         description={
           repoWithNoJob
-            ? 'Your repository is loaded, but no jobs were found.'
-            : 'Add a repository to get started.'
+            ? 'Your definitions are loaded, but no jobs were found.'
+            : 'Add a job to get started.'
```

Comment:
In situations where we refer to a repository or the workspace, use "definitions" where it makes grammatical sense.